styles.css

```diff
@@ -0,0 +1,502 @@
+/* Google Fonts Import Template Merriweather & ROBOTO */
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Roboto:wght@300;400;500;700&display=swap');
+
+/*Alternatieven voor fonts:
+-
+-
+-
+-
+-
+-
+*/
+
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Helper Classes
+    4. Media Defaults
+    5. Layout
+    6. Areas
+        6.1 Header
+        6.2 Introduction
+////////////////////////*/
+
+/*Zie resetstyles.css voor implementatie reset regels aangeduid met nummering.*/
+/* ------------------------------- 1. Global Styles -------------------------------- */
+/* Variabelen, Basale resets en standaard browserinstellingen */
+/*Zie variables.css voor toegepaste variabelen
+
+/* 1. Box-sizing reset */
+/* 2. Remove default margin */
+
+/* 3. Text rendering improvement */
+/* 4. Line height reset */
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+/* Regels die specifiek gericht zijn op typografie, zoals line-height */
+
+html {
+    font-family: var(--f1);
```

Comment:
`font-family` set je meestal in de body-selector. En is `f1` een duidelijke variabelnaam? ;)